prow/external-plugins/cherrypicker/main.go

```diff
@@ -65,6 +66,7 @@ func gatherOptions() options {
 	fs.StringVar(&o.webhookSecretFile, "hmac-secret-file", "/etc/webhook/hmac", "Path to the file containing the GitHub HMAC secret.")
 	fs.BoolVar(&o.prowAssignments, "use-prow-assignments", true, "Use prow commands to assign cherrypicked PRs.")
 	fs.BoolVar(&o.allowAll, "allow-all", false, "Allow anybody to use automated cherrypicks by skipping GitHub organization membership checks.")
+	fs.BoolVar(&o.issueOnFailure, "create-issue-on-failure", false, "Create a GitHub issue and assign it to the requestor if a cherrypick fails.")
```

Comment:
There are other reasons why a cherrypick can fail (eg. flaky Github API) so maybe make it clearer here that this is specifically for conflicts in the cherrypick?

Reply:
Makes sense. I renamed this flag to `--create-issue-on-conflict` and updated the help text accordingly.